src/edu/csus/ecs/pc2/core/list/TeamAccountComparator.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.list;
+
+import java.io.Serializable;
+import java.util.Comparator;
+
+import edu.csus.ecs.pc2.core.StringUtilities;
+import edu.csus.ecs.pc2.core.imports.clics.TeamAccount;
+
+/*
+ * TeamAccount Comparator, Order the accounts by client number then display name
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class TeamAccountComparator implements Comparator<TeamAccount>, Serializable {
+
+    /**
+     * 
+     */
+    private static final long serialVersionUID = 6930011395916416715L;
+
```

Comment:
This is just a personal opinion, but I think randomly-generated serialVersionUIDs are at best useless and at worst misleading.   I think the first version of a class should have serialVersionUID 1; if the class is changed so that it is no longer serialization-compatible the version should be bumped to 1, etc.   (With a serialVersionUID like 6930011395916416715L. what would be the appropriate "next serialVersionUID" value when the class is upgraded?)

Reply:
A serialVersionUID  is not useless, it is critical to support non-system-breaking changes in classes.
A UID is not necessarily a sequential id.   If is a different Id.   If there were no use for a random id why would
eclipse provide a eclipse Fix with a random UID?

Reply:
I didn't say that a serialVersionUID is useless.  I said that a _**randomly-generated**_ SerialVersionUID is useless (or worse, misleading.  I totally agree that having a SVUID is important; I just think that using random ones obfuscates one of the purposes, which is to help the programmer keep track of "versions".   If I have a SVUID of "1", and I upgrade the class and raise the SVUID to "2", that makes if very easy for others to know "which came first".   If I have a SVUID of 6930011395916416715L and I upgrade the class and change it to 6730411565912316751L, a programmer looking at those two numbers learns _nothing_ about the history of the revision sequence.  That's all I'm saying.  You said "A UID is not necessarily a sequential id. If is a different Id. "  That's my point:  making it different _and sequential_ provides additional information that's lost if it's _just different_.

As for why Eclipse provides a random UID, who knows?   I certainly would be hesitant to use "because Eclipse does it that way" as a justification for _anything_  :)

In any case, as I said in the opening line, "this is just a personal opinion".  If you agree, please change the UID and mark this comment resolved.  If you disagree, please leave the UID unchanged and likewise mark the comment resolved.